modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -100,8 +100,8 @@
             }
             $google_play_accounts_config_content = {
                 'dep' => {
-                  'service_account' => 'dummy',
-                  'certificate' => 'dummy',
+                  'service_account' => $google_play_config['dep']['service_account'],
```

Comment:
Indeed, the value is already defined to be "dummy" here https://github.com/mozilla-releng/build-puppet/pull/323/files#diff-aa7dacd4373035e75425f8728aca336dL96
I'm sorry, I should have double checked before asking https://github.com/mozilla-releng/build-puppet/pull/317#discussion_r237870273

Reply:
No worries, it was a team-effort "oops" :)